### PR TITLE
Some readme improvements

### PR DESCRIPTION
**Pull request by hfjn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,28 +165,18 @@ You can then run the integration tests against this stack with `pytest tests/ --
 
 Alternatively you can go the fast way and just run the whole stack + integration tests in docker: 
 
-```
+```bash
 make integration--test
```
Comment (ognjen-j) [inline README.md]:
```suggestion
make integration-test
```

File: README.md [hunk 2]
```diff
@@ -126,14 +141,22 @@ topics:
 
 To setup your development environment, make sure you have at least Python 3.6 & [poetry](https://github.com/sdispater/poetry) installed, then run 
 
-```
-poetry install --develop DEVELOP
+```bash
+poetry install --develop esque
```
Comment (swenzel) [inline README.md]: poetry installs development packages by default.
```suggestion
poetry install
```

File: README.md [hunk 3]
```diff
@@ -126,14 +141,22 @@ topics:
...
+
+```bash
+pip install pre-commit
+pre-commit install
+pre-commit install-hooks
+```
+
+### Run tests
 
 #### Integration Tests
```
Comment (swenzel) [inline README.md:166]: Maybe add that you need docker and docker-compose for that.

File: README.md [hunk 4]
```diff
@@ -143,28 +166,18 @@ You can then run the integration tests against this stack with `pytest tests/ --
...
-To install pre commit hooks run:
-
-```
-pip install pre-commit
-pre-commit install
-pre-commit install-hooks
-```
 ## Additional information
 
 You can find additional information, development cycle and release information on our [wiki page](https://github.com/real-digital/esque/wiki).
```
Comment (swenzel) [inline README.md]: Do we need to update the wiki as well?

File: README.md [hunk 5]
```diff
@@ -13,88 +13,103 @@ Some stuff is hard, and that is okay, but listing your kafka topics shouldn't be
...
 * insightful
-* made by engineers for engineers
+* by engineers for engineers
 
-## Basic Features
+## Feature Overview
 
 * Support for any type of Kafka deployment >1.2
-* List Resources (Topics, Consumergroups, Brokers)
+* Listing of Resources (Topics, Consumergroups, Brokers)
```
Comment (garrettthomaskth) [inline README.md]: All the other sentences start with a verb. What about "List Resource Instances" or "Display Resources"

File: README.md [hunk 6]
```diff
@@ -13,88 +13,103 @@ Some stuff is hard, and that is okay, but listing your kafka topics shouldn't be
...
+  --recreate-config  Overwrites the config with the sample config.
+  --version          Show the version and exit.
+  -v, --verbose
+  --no-verify        Skips all verification dialogs and answers them with yes.
+  --help             Show this message and exit.
 
-Autocompletion is automatically installed via a post-install hook in the setup.py. 
-If it doesn't work for some reason you can still install it yourself: 
+Commands:
+  apply     Apply a configuration
```
Comment (garrettthomaskth) [inline README.md]:
```suggestion
  apply     Apply a set of topic configurations.
```

File: README.md [hunk 7]
```diff
@@ -13,88 +13,103 @@ Some stuff is hard, and that is okay, but listing your kafka topics shouldn't be
...
+  config    Configuration-related options
```
Comment (garrettthomaskth) [inline README.md]:
```suggestion
  config    Configuration-related options.
```

File: README.md [hunk 8]
```diff
@@ -13,88 +13,103 @@ Some stuff is hard, and that is okay, but listing your kafka topics shouldn't be
...
+  consume   Consume messages of a topic from one environment to a file or...
```
Comment (garrettthomaskth) [inline README.md]:
```suggestion
  consume   Consume messages from a topic.
```

File: README.md [hunk 9]
```diff
@@ -13,88 +13,103 @@ Some stuff is hard, and that is okay, but listing your kafka topics shouldn't be
...
+* Consume and Produce to and from Avro and Plaintext Topics (including Avro Schema Resolution from Schema Registry)
 * Context Switch (Easily Switch between pre-defined Clusters)
 * Kafka Ping (Test roundtrip time to your kafka cluster)
 
-## Installation and Usage
+## Command Overview
 
-### Installation
+```bash
+$ esque
```
Comment (garrettthomaskth) [inline README.md:45]: Usage: esque [OPTIONS] COMMAND [ARGS]...

  esque - an operational kafka tool.

  In the Kafka world nothing is easy, but esque (pronounced esk) is an
  attempt at it.

Options:
  --recreate-config  Overwrites the config with the sample config.
  --version          Show the version and exit.
  -v, --verbose      Return stack trace on error.
  --no-verify        Skip all verification dialogs and answer them with yes.
  --help             Show this message and exit.

Commands:
  apply     Apply a set of topic configurations.
  config    Configuration-related options.
  consume   Consume messages from a topic.
  create    Create a new instance of a resource.
  ctx       List contexts and switch between them.
  delete    Delete a resource.
  describe  Get detailed information about a resource.
  edit      Edit a resource.
  get       Get a quick overview of different resources.
  ping      Test the connection to the kafka cluster.
  produce   Produce messages to a topic.

File: README.md [hunk 10]
```diff
@@ -13,88 +13,103 @@ Some stuff is hard, and that is okay, but listing your kafka topics shouldn't be
...
+```bash
 No config provided in ~/.esque
 Should a sample file be created in ~/.esque [y/N]:
 ```
 
-When answering with `y` `esque` will copy over the [sample config](https://github.com/real-digital/esque/blob/master/sample_config.cfg) to `~/.esque/esque.cfg`.
+When answering with `y` `esque` will copy over the [sample config](https://github.com/real-digital/esque/blob/master/esque/config/sample_config.yaml) to `~/.esque/esque.yaml`.
 Afterwards you can modify that file to fit your cluster definitions.
 
 Alternatively might just provide a config file following the sample config's file in that path.
```
Comment (garrettthomaskth) [inline README.md:100]:
```suggestion
Alternatively can provide your own `esque_conf.yaml` config file in `~/.esque/`
```

File: README.md [hunk 11]
```diff
@@ -13,88 +13,103 @@ Some stuff is hard, and that is okay, but listing your kafka topics shouldn't be
...
 When starting `esque` for the first time the following message will appear:
 
-```
+```bash
 No config provided in ~/.esque
 Should a sample file be created in ~/.esque [y/N]:
 ```
 
-When answering with `y` `esque` will copy over the [sample config](https://github.com/real-digital/esque/blob/master/sample_config.cfg) to `~/.esque/esque.cfg`.
+When answering with `y` `esque` will copy over the [sample config](https://github.com/real-digital/esque/blob/master/esque/config/sample_config.yaml) to `~/.esque/esque.yaml`.
```
Comment (garrettthomaskth) [inline README.md]: is it esque.yaml or esque_config.yaml now?

File: README.md [hunk 12]
```diff
@@ -13,88 +13,103 @@ Some stuff is hard, and that is okay, but listing your kafka topics shouldn't be
...
+## Feature Overview
 
 * Support for any type of Kafka deployment >1.2
-* List Resources (Topics, Consumergroups, Brokers)
+* Listing of Resources (Topics, Consumergroups, Brokers)
 * Get detailed Overviews of Resources (Topics, Consumergroups, Brokers)
 * Create/Delete Topics
+* Edit Topic Configurations
+* Edit Consumer Offset for Topics
+* SASL/SSL Support support out of the box
```
Comment (garrettthomaskth) [inline README.md]:
```suggestion
* SASL/SSL Support out of the box
```

File: README.md [hunk 13]
```diff
@@ -13,88 +13,103 @@ Some stuff is hard, and that is okay, but listing your kafka topics shouldn't be
...
-* made by engineers for engineers
+* by engineers for engineers
 
-## Basic Features
+## Feature Overview
 
 * Support for any type of Kafka deployment >1.2
-* List Resources (Topics, Consumergroups, Brokers)
+* Listing of Resources (Topics, Consumergroups, Brokers)
 * Get detailed Overviews of Resources (Topics, Consumergroups, Brokers)
```
Comment (garrettthomaskth) [inline README.md]:
```suggestion
* Get detailed Overviews of Resources (Topics, Consumer Groups, Brokers)
```

File: README.md (hunk 5 above)
Comment (garrettthomaskth) [inline README.md]:
```suggestion
* Listing of Resources (Topics, Consumer Groups, Brokers)
```

File: README.md [hunk 14]
```diff
@@ -143,28 +166,18 @@ You can then run the integration tests against this stack with `pytest tests/ --
...
 
-```
-make integration--test
+```bash
+make integration-test
 ```
 
 #### Unit Tests
 
 If you just want the unit tests simply run:
```
Comment (garrettthomaskth) [inline README.md]:
```suggestion
If you only want the unit tests, just run:
```

File: README.md [hunk 15]
```diff
@@ -108,7 +123,7 @@ changed and specified.
 It has to use the schema, which is used
```
Comment (garrettthomaskth) [inline README.md]:
```suggestion
It has to use the same schema, which is used 
```

File: README.md (hunk 4 above)
Comment (hfjn) [reply README.md]: Actually I was about to suggest to drop the wiki. Currently it holds no information which isn't in the readme. What do you think @ognjen-j ?

File: README.md (hunk 5 above)
Comment (hfjn) [reply README.md]:
```suggestion
* Display Resources (Topics, Consumergroups, Brokers)
```

File: README.md (hunk 11 above)
Comment (hfjn) [reply README.md]: I have an `esque.yaml` locally. :)

File: README.md [hunk 16]
```diff
@@ -13,88 +13,108 @@ Some stuff is hard, and that is okay, but listing your kafka topics shouldn't be
...
 When starting `esque` for the first time the following message will appear:
 
-```
+```bash
 No config provided in ~/.esque
 Should a sample file be created in ~/.esque [y/N]:
 ```
 
-When answering with `y` `esque` will copy over the [sample config](https://github.com/real-digital/esque/blob/master/sample_config.cfg) to `~/.esque/esque.cfg`.
+When answering with `y` `esque` will copy over the [sample config](https://github.com/real-digital/esque/blob/master/esque/config/sample_config.yaml) to `~/.esque/esque.yaml`.
```
Comment (swenzel) [inline README.md]:
```suggestion
When answering with `y` `esque` will copy over the [sample config](https://github.com/real-digital/esque/blob/master/esque/config/sample_config.yaml) to `~/.esque/esque_config.yaml`.
```

**Review by swenzel — APPROVED** [review]

**Pull request merged**